lib/discourse_webauthn.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 require "webauthn/challenge_generator"
-require "webauthn/security_key_base_validation_service"
-require "webauthn/security_key_registration_service"
-require "webauthn/security_key_authentication_service"
+require "webauthn/base_validation_service"
+require "webauthn/registration_service"
+require "webauthn/authentication_service"
```

Comment:
Kind of odd that we need to manually require stuff here. I'm guessing this is because we're not following zeitwerk conventions here? Perhaps we just rename the path from `webauthn` to `discourse_webauthn`?

Reply:
Yes, renaming the path seems to work.